### ISSUE TITLE
Testing re-open

### ISSUE THREAD

**Issue by bmaca** [ccstolley/ghub#8]

**Comment by bmaca** [comment]
@ccstolley User gets a nasty error,if there is no body text when they open an issue. Maybe handle this more gracefully. 
```ghub -i 8
       Title : #8 Testing re-open
   Submitter : bmaca
    Assignee : None
  Created At : 2015-10-17T20:14:15Z
         URL : https://github.com/ccstolley/ghub/issues/8
Traceback (most recent call last):
  File "/usr/local/bin/ghub", line 9, in <module>
    load_entry_point('ghub==1.0', 'console_scripts', 'ghub')()
  File "build/bdist.macosx-10.10-intel/egg/ghub.py", line 549, in main
  File "build/bdist.macosx-10.10-intel/egg/ghub.py", line 270, in display_issues
  File "build/bdist.macosx-10.10-intel/egg/ghub.py", line 354, in print_pull_request
AttributeError: 'NoneType' object has no attribute 'splitlines'````

**Comment by ccstolley** [comment]
Resolved by #9
